randovania/games/dread/gui/ui_files/games_tab_dread_widget.ui

```diff
@@ -183,80 +198,27 @@
ing>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Metroid Dread is known to crash in some circumstances. Make sure to read the following list and how to avoid these.&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; text-decoration: underline;&quot;&gt;After using Warp to Start in a Save Station&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Occasionally after using warp to start, the game crashes at the end of the loading screen. It is not known what circumstances cause the warp to crash.&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;Workaround&lt;/span&gt;: Always save before using warp to start so no progression is lost. After the crash, try again.&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; text-decoration: underline;&quot;&gt;After collecting a suit upgrade&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Changing the suit model while a visual effect is present on Samus crashes the game, and the certain room transitions causes the suit model to update. The following situations are triggers visual effects:&lt;/p&gt;&lt;ul style=&quot;margin-top: 0px; margin-bottom: 0px; margin-left: 0px; margin-right: 0px; -qt-list-indent: 1;&quot;&gt;&lt;li style=&quot; margin-top:12px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Charging a spark or shinesparking&lt;/li&gt;&lt;li style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Charging a Power Bomb&lt;/li&gt;&lt;li style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Being in a heated/cold room&lt;/li&gt;&lt;li style=&quot; margin-top:0px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Using any Aeion ability&lt;/li&gt;&lt;/ul&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;Workaround&lt;/span&gt;: Manually update your suit model by pressing ZL+ZR+D-Pad Up in a known safe state.&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; text-decoration: underline;&quot;&gt;Random crashes in Ryujinx&lt;/span&gt;&lt;/p&gt;&lt;p&gt;It has been reported that Metroid Dread in Ryujinx would crash at any point, with no indication, when using the Memory Manager Mode &amp;quot;Host Unchecked&amp;quot;.&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;Workaround&lt;/span&gt;: In Ryujinx configuration, do not use &amp;quot;Host Unchecked&amp;quot;. &lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; text-decoration: underline;&quot;&gt;Crash in Waterfall room in Artaria&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Rare crashes in the Waterfall room in West Artaria has been reported when playing on console. No reason has been found.&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;Workaround&lt;/span&gt;: Save in the Save Station immediately west of Red Teleportal.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```

Comment:
after the last commit, did you double check that these still look proper?
Because you now have HTML text, but are telling QT that you are using markdown.